examples/user_guide/Custom_Components.ipynb

```diff
@@ -349,7 +359,7 @@
     "    _template = \"\"\"\n",
     "    <select id=\"select\" value=\"${value}\" style=\"width: ${model.width}px\">\n",
     "      {% for option in options %}\n",
-    "      <option id=\"option\">${option}</option>\n",
+    "      <option id=\"option-{{ loop.index0 }}\">{{ option }}</option>\n",
```

Comment:
I don't understand why you cannot use `id='option'`. But it raises an exception.

Reply:
You absolutely should be able to as it should automatically add the index.